### PR TITLE
Improve Transform docs

### PR DESCRIPTION
**Pull request by chriswhong**
- Renames `Starlark` top-level docs  section to `Transforms`
- Renames `introduction` to `Transforms Overview`
- Adds `Transforms Quickstart` with commands for creating a new dataset version from a CSV download
- Improves `Example Transforms` with a overview paragraph of what each example does before showing the code
- Reorders Transforms docs pages to prioritize overview and quickstart, moves reference materials to the bottom of the list
- Adds server-side redirects for existing `docs/starlark` routes

Example_Transforms___Qri_io

### CODE REVIEW

File: content/docs/transforms/examples.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+  This documentation is for qri CLI (command line) only. Composing Transform scripts on Desktop is a work in progress.
+</InfoBlock>
+
+Transforms allow you to bind data with code in Qri, providing a quick means to automate the creation of new dataset versions from websites, APIs, and _other qri datasets_.
+
+To get started with transforms, check out [Transforms Qruickstart](https://qri.io/docs/transforms/quickstart).
+
+Below are several examples of transform scripts that demonstrate various approaches to automating qri datasets.  If you have a transform script that you think might help others, consider [opening a pull request](https://github.com/qri-io/website/pulls) to add it here.
+
+-- --
```
Comment (b5) [inline content/docs/transforms/examples.md]: I'd love to flip the order of these from less-to-more complex. What about:
* MTA Elevator Changes
* 311 Complaints By Boro
* World Bank Population

MTA should work in isolation, 311 involves geospatial calculations & a _big_ API fetch, and the last one requires you already have the `b5/country_codes` dataset before running

File: content/docs/transforms/examples.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+## Count NYC 311 Complaints By Boro (Spatial Join and Aggregation)
+
+### Overview
```
Comment (b5) [inline content/docs/transforms/examples.md]: Can we drop the `### Overview` and `### Script` headers? They have the same font size as the section titles when rendered, making the page a little harder to read

File: content/docs/transforms/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
+---
+metaTitle: "Transforms Quickstart"
+metaDescription: "Quickly get started with transforms using Qri CLI"
+weight: 2
+---
+
+import InfoBlock from '../../../src/components/InfoBlock.js'
+
+This docs page will give you example commands to get started with qri transform scripts using qri CLI.
```
Comment (b5) [inline content/docs/transforms/quickstart.md]:
```suggestion
Follow these steps to run your first transform script with qri CLI.
```

File: content/docs/transforms/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+import InfoBlock from '../../../src/components/InfoBlock.js'
+
+This docs page will give you example commands to get started with qri transform scripts using qri CLI.  
+
+## Create a Directory to Work From
+
+Create an empty directory using `mkdir`, `cd` into the new directory.
+
+```bash
+[~/datasets]$ mkdir example-transform
```
Comment (b5) [inline content/docs/transforms/quickstart.md]: I find the directory name in the prompt a little confusing. Can we drop them in favor of just `$` to mean "things you should type"?

File: content/docs/transforms/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+```
+
+## Add Your Transform Code
+
+Open `transform.star` in your favorite text editor, and add some code.  You can copy and paste this simple example that downloads a CSV from the web and sets a dataset's `body`.
+
+```python
+load("http.star", "http")
+
+def download(ctx):
```
Comment (b5) [inline content/docs/transforms/quickstart.md]:
```suggestion
# qri will call the download function first
def download(ctx):
```

File: content/docs/transforms/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+  # Download CSV of all earthquakes in the past 7 days from the U.S. Geological Survey
+  res = http.get("https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_week.csv")
+
+  # res.body() yields the CSV as a string
+  return res.body()
+
+def transform(ds, ctx):
```
Comment (b5) [inline content/docs/transforms/quickstart.md]:
```suggestion
# all transform scripts have a transform function, qri calls transform after download
def transform(ds, ctx):
```

File: content/docs/transforms/quickstart.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    "syntaxVersion": "0.9.6-dev"
+  },
+  "viz": {
+    "format": "html",
+    "qri": "vz:0",
+    "renderedPath": "/map/QmXpCuUQ4bHwzphGztd5XExCokLtdk5rSZDT7CGvVwPFeU",
+    "scriptPath": "/map/QmZkRmrbqHnw8zs48ApogHswSWPXwg5Qc2TT5ZdKYUiJrH"
+  }
+}
+[~/datasets/example-transform]$
```
Comment (b5) [inline content/docs/transforms/quickstart.md]: drop empty prompt

File: content/docs/transforms/quickstart.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+1   fred/earthquakes
+    /ipfs/QmSx8CKD2ojZyhyWB1s4RXAcPeM7bUeKTjrsxP55adYXnr
+    435 kB, 2329 entries, 3012 errors
+
+(END)
+```
+
+## Save Again in the Future
+
+Each time you run the above `qri save` command, the transform script will pull the CSV of earthquakes.  If the downloaded CSV is different from the latest version of the dataset, qri will add a new version that supersedes it.
```
Comment (b5) [inline content/docs/transforms/quickstart.md]: were you able to run `qri save  fred/earthquakes --recall transform` to re-execute the script? The upside of recall: the code is now bound to the dataset, you don't need to keep track of the file.

File: content/docs/transforms/starlib.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
-metaTitle: "Standard Library"
+metaTitle: "Starlark Standard Library"
```
Comment (b5) [inline content/docs/transforms/starlib.md]: Ah I also forgot, the comment on this page is real, it's _actually_ generated by a script & template in the `scripts` folder of this repo. To make changes to this, at least edit the template as well so these changes aren't overwritten.

File: content/docs/transforms/examples.md (hunk 1 above)
Comment (chriswhong) [reply content/docs/transforms/examples.md]: Great idea, done

File: content/docs/transforms/examples.md (hunk 2 above)
Comment (chriswhong) [reply content/docs/transforms/examples.md]: For sure, was planning to have more sections but ended up with just one intro paragraph and the code snippet.

File: content/docs/transforms/quickstart.md (hunk 4 above)
Comment (chriswhong) [reply content/docs/transforms/quickstart.md]: 👍 Done.

File: content/docs/transforms/quickstart.md (hunk 7 above)
Comment (chriswhong) [reply content/docs/transforms/quickstart.md]: Done

File: content/docs/transforms/quickstart.md (hunk 8 above)
Comment (chriswhong) [reply content/docs/transforms/quickstart.md]: Added a bit about `--recall` at the bottom of quickstart.

File: content/docs/transforms/starlib.md (hunk 9 above)
Comment (chriswhong) [reply content/docs/transforms/starlib.md]: 👍

**Pull request merged**